code/HabitTracker/app/src/main/java/com/example/habittracker/activities/eventlist/LocationActivity.java

```diff
@@ -0,0 +1,61 @@
+package com.example.habittracker.activities.eventlist;
```

Comment:
This whole file needs java doc comments. Please add.

Reply:
Fixed